src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -189,7 +193,22 @@
                                        FontSize="16"
                                        HorizontalAlignment="Right"
                                        Margin="0,0,36,0"
-                                       FontFamily="Segoe MDL2 Assets" />
+                                       FontFamily="Segoe MDL2 Assets" /> -->
+
+                            <Button Content="&#xE971;"
+                                    Grid.Row="0"
+                                    HorizontalAlignment="Right"
+                                    Margin="0,0,36,0"
+                                    Background="Transparent"
```

Comment:
Will this play nice with a localized / longer ToggleSwitch label? If not - I'd suggest turning those labels of by overriding the OnContent and OffContent - the ToggleSwitch visual should be enough.

Reply:
The button is right next to the localized text in Russian. It's possible that in some localizations text would be longer. So turning those labels off is a good idea. 

![image](https://user-images.githubusercontent.com/8949536/112167987-f9e01400-8c01-11eb-8939-5ccd6b6fa8fa.png)